.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: corretto
+      - name: Build
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: build
+          gradle-executable: ./gradlew
```

Comment:
Why is this needed? It is said [here](https://github.com/gradle/gradle-build-action#using-a-specific-gradle-executable) that this executable should be the default